conjure-java-core/src/main/java/com/palantir/conjure/java/services/UndertowServiceHandlerGenerator.java

```diff
@@ -349,6 +352,28 @@ private TypeSpec generateEndpointHandler(
         return endpointBuilder.build();
     }
 
+    private static void addTags(EndpointDefinition endpointDefinition, TypeSpec.Builder endpointBuilder) {
+        if (!endpointDefinition.getTags().isEmpty()) {
+            CodeBlock arrayValues = CodeBlock.join(
+                    Collections2.transform(endpointDefinition.getTags(), value -> CodeBlock.of("$S", value)), ", ");
```

Comment:
Hmm, we already use guava ListenableFutures in generated code, we might as well use ImmutableList here.

Reply:
Updated the endpoints list as well, I'm a lot happier with how it looks :-)